src/suricata.c

```diff
@@ -2358,12 +2358,11 @@ static int StartInternalRunMode(SCInstance *suri, int argc, char **argv)
 
 static int FinalizeRunMode(SCInstance *suri, char **argv)
 {
-    switch (suri->run_mode) {
-        case RUNMODE_UNKNOWN:
-            PrintUsage(argv[0]);
-            return TM_ECODE_FAILED;
-        default:
-            break;
+    if (suri->run_mode == RUNMODE_UNKNOWN) {
+        SCLogError("No capture mode specified as a command-line argument, specify one from the "
```

Comment:
I find the output of `--list-runmodes` to be pretty much useless right now, so I think that needs to be fixed first.

Reply:
To add more context to this comment: the contributions in this PR are important, but before we can move on with them, we'll have to rethink what we want the `--list-runmodes` output to look like. We'll try to tackle that in the next couple days, then we'll get back to this task. :)

Reply:
> To add more context to this comment: the contributions in this PR are important, but before we can move on with them, we'll have to rethink what we want the `--list-runmodes` output to look like. We'll try to tackle that in the next couple days, then we'll get back to this task. :)

Okay

Reply:
@jufajardini is there another redmine ticket to link to https://redmine.openinfosecfoundation.org/issues/5711 (as it is blocking 5711) ?

Reply:
@catenacyber Do you mean a ticket to refactor `--list-runmodes`? There is, now: https://redmine.openinfosecfoundation.org/issues/6572

Reply:
yes

Reply:
I've updated the ticket to add a suggestion.